Project/QtCreator/qctools-gui/qctools-gui.pro

```diff
@@ -48,7 +50,11 @@ HEADERS += \
     $$SOURCES_PATH/GUI/Imagelabel.h \
     $$SOURCES_PATH/GUI/config.h \
     $$SOURCES_PATH/GUI/draggablechildrenbehaviour.h \
-    $$SOURCES_PATH/ThirdParty/cqmarkdown/CMarkdown.h
+    $$SOURCES_PATH/ThirdParty/cqmarkdown/CMarkdown.h \
+    $$SOURCES_PATH/GUI/booleanchartconditioneditor.h \
```

Comment:
Suggest an overall adjustment from `booleanchart*` to `barchart*` and to generally phase out the term `boolean` from this PR.

Reply:
ok.

Reply:
done